test/MainEvent.cxx

```diff
@@ -174,6 +186,7 @@ int main(int argc, char **argv)
       Int_t nentries = (Int_t)tree->GetEntries();
       nevent = TMath::Min(nevent,nentries);
       if (read == 1) {  //read sequential
+         ioperf = getenv("ENABLE_TTREEPERFSTATS") ? new TTreePerfStats("Perf Stats", tree) : nullptr;
```

Comment:
this is the last change need for merging.  [Adding support for configure/make so that we can backport to older version can/should be another merge request]